### PR TITLE
Remove viz components in control plane dashboard

### LINKED ISSUE #5898: 2.10 dashboard doesn't show all components on control panel
### What is the issue?
In the control panel overview of the dashboard it show all the involved deployments and the number of pods.
With the separation the components involved that are in the linkerd-viz namespace aren't showing up anymore.

### Logs, error output, etc

Screenshot 2021-03-14 at 18 28 17

### Environment

- Kubernetes Version: 1.19.7
- Cluster Environment: (GKE, AKS, kops, ...)
- Host OS:
- Linkerd version: 2.10.0 ( upgrade from 2.9.4 )

### PR DESCRIPTION
**Pull request by dadjeibaah**
This change modifies the `/controlplane` dashboard page by removing viz
components that are no longer part of the core control plane install. It
also adds `Proxy Injector` to the list of components since that was
missing from the list.

Fixes #5898

Signed-off-by: Dennis Adjei-Baah <dennis@buoyant.io>

image

### CODE REVIEW

File: web/app/js/components/ServiceMesh.jsx
Comment (dadjeibaah) [inline web/app/js/components/ServiceMesh.jsx:238]: The property `api` was unused so I removed this as cleanup.

**Review by adleong — APPROVED** [review]

**Review by Pothulapati — APPROVED** [review]
LGTM

**Review by kleimkuhler — APPROVED** [review]
Looks good. I'd keep an eye on #5936 which will require removing the SP validator from here as well. I think that PR is just waiting to merge until after `2.10.1` is released, so maybe take care of it as part of this change (once that merges)?

**Comment by dadjeibaah** [comment]
Good point @kleimkuhler. I'm cool with waiting for this to merge until after #5936.

**Comment by kleimkuhler** [comment]
After some off GH discussion I'm merging this to include in the edge. We'll have a follow-up after #5936 to remove SP validator.

**Pull request merged**
